examples/image.rs

```diff
@@ -71,46 +71,132 @@ fn start_example_client_code() -> Result<()> {
+        "entity_type": "image",
+        "asset_type": "svg",
+        "image_id": "example_svg",
+        "url": "https://www.svgrepo.com/show/369528/rust.svg",
+        "resolution": { "width": VIDEO_RESOLUTION.width, "height": VIDEO_RESOLUTION.height },
+
+    }))?;
+    common::post(&json!({
+        "type": "register",
+        "entity_type": "image",
+        "asset_type": "png",
+        "image_id": "example_png",
+        "url": "https://rust-lang.org/logos/rust-logo-512x512.png",
+        "resolution": { "width": VIDEO_RESOLUTION.width, "height": VIDEO_RESOLUTION.height },
+
     }))?;
 
     info!("[example] Update scene");
     common::post(&json!({
         "type": "update_scene",
         "inputs": [],
         "nodes": [
-           {
-                "node_id": "static_image",
-                "type": "image",
-                "image_id": "example_image",
-           }
+            {
+                 "node_id": "gif_1",
+                 "type": "image",
+                 "image_id": "example_gif",
+            },
+            {
+                 "node_id": "png_1",
+                 "type": "image",
+                 "image_id": "example_png",
+            },
+            {
+                 "node_id": "jpeg_1",
+                 "type": "image",
+                 "image_id": "example_jpeg",
+            },
+            {
+                 "node_id": "svg_1",
+                 "type": "image",
+                 "image_id": "example_svg",
+            },
+            {
+                "node_id": "gif_1_rescaled",
+                "type": "built-in",
+                "transformation": "transform_to_resolution",
+                "strategy": "fit",
+                "resolution": { "width": 960, "height": 540 },
+                "input_pads": ["gif_1"],
+                "background_color_rgba": "#00000000",
+            },
+            {
+                "node_id": "png_1_rescaled",
+                "type": "built-in",
+                "transformation": "transform_to_resolution",
+                "strategy": "fit",
+                "resolution": { "width": 960, "height": 540 },
+                "input_pads": ["png_1"],
+                "background_color_rgba": "#00000000",
+            },
+            {
+                "node_id": "jpeg_1_rescaled",
+                "type": "built-in",
+                "transformation": "transform_to_resolution",
+                "strategy": "fit",
+                "resolution": { "width": 960, "height": 540 },
+                "input_pads": ["jpeg_1"],
+                "background_color_rgba": "#00000000",
+            },
+            {
+                "node_id": "svg_1_rescaled",
+                "type": "built-in",
+                "transformation": "transform_to_resolution",
+                "strategy": "fit",
+                "resolution": { "width": 960, "height": 540 },
+                "input_pads": ["svg_1"],
+                "background_color_rgba": "#00000000",
```

Comment:
You don't have to specify this. It's default value

Reply:
> Received malformed request:\nmissing field `background_color_rgba`"

Reply:
LOL I will fix it